composer.json

```diff
@@ -29,6 +28,13 @@
     }
   },
   "require-dev": {
+    "phpunit/phpunit": "^5.2",
```

Comment:
Not sure about this; many of the actual (not test) classes depend on PHPUnit classes, so PHPUnit is actually a hard dependency of this package (which would typically be included as a dev dependency itself).